hair_salon/data/base_automation.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
```

Comment:
Don't we need a filter_pre_domain? So that it is not run as soon as we touch any calendar.event in the db.